packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -2687,6 +2688,18 @@ function warnAboutUpdateOnUnmountedFiberInDEV(fiber) {
       // Only warn for user-defined components, not internal ones like Suspense.
       return;
     }
+
+    if (
+      deferPassiveEffectCleanupDuringUnmount &&
+      runAllPassiveEffectDestroysBeforeCreates
+    ) {
+      // If there are pending passive effects unmounts for this Fiber,
+      // we can assume that they would have prevented this update.
+      if (pendingPassiveHookEffectsUnmount.includes(fiber)) {
+        return;
+      }
+    }
+
```

Comment:
odd, syntax highlighting seems to be broken for this file 
![image](https://user-images.githubusercontent.com/18808/75060921-999b1200-54d7-11ea-9929-0fd5c690e82c.png)

Reply:
Huh. Yeah for me me too.